src/explore-education-statistics-admin/src/pages/release/data/components/__tests__/ApiDataSetCreateModal.test.tsx

```diff
@@ -29,6 +29,8 @@ describe('ApiDataSetCreateModal', () => {
   ];
 
   test('renders warning message in modal when no candidates', async () => {
+    const history = createMemoryHistory();
```

Comment:
If we don't navigate into the link, we don't need to check the history and can remove this (and it's usage below)

Reply:
As above